### PR TITLE
Add eval command

### LINKED ISSUE #275: "JS" command in Terminal !
**I have an idea for a new feature , I don't know if it's really feasible! The feature is simple:** When you note `JS` in the terminal , it launches you a JavaScript console like the one present in the developer option _(F12)_ . We could then use the javascript console to perform actions not possible the base .

### PR DESCRIPTION
**Pull request by hello-smile6**
Closes #275

### CODE REVIEW

File: src/containers/applications/apps/terminal.js [hunk 1]
```diff
@@ -50,6 +50,11 @@ export const WnTerminal = ()=>{
       }else{
         tmpStack.push("ECHO is on.");
       }
+    }
+    else if(type=="eval"){
+      if(arg.length) {
+        tmpStack.push(eval(arg).toString());
+      }
```
Comment (hello-smile6) [inline src/containers/applications/apps/terminal.js:57]: Ugh, need another `}`

**Comment by hello-smile6** [comment]
> ⌛ Deploy Preview - Build in Progress If you are part of our discord server then leave your discord name for a role.

@win11bot needs to rebuild!!

**Comment by Posandu** [comment]
@hello-smile6 commit again

**Comment by hello-smile6** [comment]
> ❌ DEPLOY PREVIEW BUILD HAS FAILED

DUH! Rebuild!

> @hello-smile6 commit again

I know!

**Comment by hello-smile6** [comment]
> ✔️ Deploy Preview for win11blue ready! 😎 Browse the preview: https://277.pr.blueedge.me ! 🔨 If you experience an SSL issue then wait 2 minutes and try again. 🔍 Inspect the deploy log: https://github.com/blueedgetechno/win11React/actions/runs/1759812051

Thanks! BTW, it works!
Recording 2022-01-27 at 20 16 50

**Comment by hello-smile6** [comment]
> @hello-smile6 commit again

Does it work?

**Comment by Posandu** [comment]
works

**Comment by hello-smile6** [comment]
> works

Could I add output?

**Comment by blueedgetechno** [comment]
hmm, doesn't it poses a security risk for remote code execution?

**Comment by hello-smile6** [comment]
> hmm, doesn't it poses a security risk for remote code execution?

If someone's willingly using it, they could just use a bookmarklet instead. It'd be so that people could quickly test a UI change.

**Comment by blueedgetechno** [comment]
@andrewstech what's your take on this?

**Comment by hello-smile6** [comment]
> @andrewstech what's your take on this?
> 
> Also read this first: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#never_use_eval! It is on official mozilla website.

Huh, would you be okay with executing JS in a VM?

**Comment by blueedgetechno** [comment]
What exactly do you mean by VM?

**Comment by hello-smile6** [comment]
> What exactly do you mean by VM?

https://github.com/copy/v86 (libv86)
It's decently easy to use. You could run MS-DOS in the background for the terminal.

**Comment by blueedgetechno** [comment]
what are the performance impact of it?

**Comment by hello-smile6** [comment]
> what are the performance impact of it?

Can crash an iPad Mini 2, but you could avoid using it if the device isn't fast enough. Additionally, if you pause the VM when it's not actively doing anything, it would be almost impossible to tell it's running.

**Comment by andrewstech** [comment]
> @andrewstech what's your take on this?
> 
> Also read this first: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#never_use_eval! It is on official mozilla website.

Yes it could lead to remote code execution. However as our project is static we should be fine as we are not connecting to a server.

**Comment by blueedgetechno** [comment]
> Yes it could lead to remote code execution. However as our project is static we should be fine as we are not connecting to a server.

same thought, should we merge it?

**Comment by andrewstech** [comment]
> > Yes it could lead to remote code execution. However as our project is static we should be fine as we are not connecting to a server.
> 
> same thought, should we merge it?

Yes. however somthing is broken

**Comment by yashash1511** [comment]
> Yes. however somthing is broken

what is broken?

**Comment by MalikIdreesHasanKhan** [comment]
Is it a security flaw.

**Comment by andrewstech** [comment]
Screenshot_20220130-214057_Chrome

**Comment by hello-smile6** [comment]
Works now. @Lukas-Batema

**Comment by Lukas-Batema** [comment]
> Works now. @Lukas-Batema

*I know* 
*cough* you are welcome

**Pull request merged**